randovania/games/dread/json_data/Elun.txt

```diff
@@ -286,6 +284,8 @@ Extra - asset_id: collision_camera_003
 > Dock from Purple Drapes; Heals? False
   * Layers: default
   * Blocked Passage to Purple Drapes/Tunnel to Ammo Recharge Station
+  > Event - Elun - Ammo Recharge Room Blob
+      Lay Bomb
```

Comment:
this needs moved onto the Alcove connection to the event, not here.

Reply:
Still not quite right. Dock goes to X event trivially, X event goes to Alcove trivially, Alcove goes to blob by either shoot beam or lay bomb